src/server/index.jsx

```diff
@@ -25,9 +30,25 @@ const dataFolderToRender =
 
 const articleDataRegexPath = `${articleRegexPath}.json`;
 
+logger.debug(
+  `Application outputting logs to directory "${process.env.SIMORGH_LOG_DIR}"`,
```

Comment:
Does this line appear in the journald output? If not, I'm not sure I see what it's achieving, as you'd have to be looking at the file it's describing in the first place. I'm not sure Winston has a journald output by default. 🤔

Reply:
Discussed IRL, this doesnt log in the journald output, however this is just a nice to have to local dev

Reply:
It's mainly for future users who log onto an instance on TEST or LIVE & have little experience of where the logs live -- their first step might well be to use `journalctl` to check our service's output.

But yeah -- agreed to consider in follow-up work.